testing/tests/DevExpress.exporter/exceljsParts/exceljs.tests.js

```diff
@@ -6477,6 +6478,72 @@ QUnit.module('_getFullOptions', moduleConfig, () => {
     });
 });
 
+QUnit.module('Deprecated warnings', moduleConfig, () => {
+    QUnit.test('CustomizeCell handler - warnings when \'cell\' field is not used', function(assert) {
+        assert.expect(3);
+        const stub = sinon.stub(errors, 'log', () => {
+            assert.strictEqual(true, false, 'error.log should not be called');
```

Comment:
IMHO we need this 'assert' in each test in its 'after' function:
https://github.com/DevExpress/DevExtreme/blob/20_2/testing/tests/DevExpress.exporter/exceljsParts/exceljs.tests.js#L54 - let's test the latest API only and pray that there are no issues in obsoleted

Reply:
Fixed